swmmio/core.py

```diff
@@ -212,6 +213,10 @@ def conduits(self):
         df.InletNode = df.InletNode.astype(str)
         df.OutletNode = df.OutletNode.astype(str)
 
+        tags = dataframe_from_inp(inp.path, "TAGS")
+        if "Link" in set(tags.index):
+            df = df.merge(dataframe_from_inp(inp.path, "Tags"), left_on="Name", right_on="Name", how="left")
```

Comment:
Instead of re-instantiating a tags dataframe, you could reuse the `tags` dataframe created on line 216, right? Then this line could be simplified to 

```python 
df = df.merge(tags, left_on="Name", right_on="Name", how="left")
```